.github/workflows/cache.yml

```diff
@@ -0,0 +1,26 @@
+name: Cache
+
+on:
+  push:
+    branches:
+    - master
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/cache@v1.1.2
+      id: cache
+      if: startsWith(runner.os, 'Linux')
+      with:
+        path: ~/.cache/pip
+        key: ${{ runner.os }}-pip-${{ hashFiles('**/pip-dep/requirements*.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
```

Comment:
I think we can leave off this optional key. The docs say:
> restore-keys: Optional An ordered list of alternative keys to use for finding the cache if no cache hit occurred for key.

If we don't have a cache hit, it's okay to just install the dependencies again. (In fact, that's what we want.)